meson.build

```diff
@@ -14,7 +14,9 @@ add_project_arguments(
     '-Wstrict-prototypes',
     '-Wmissing-prototypes',
     '-Wredundant-decls',
-    '-Wshadow'
+    '-Wshadow',
+    '-Wno-error=discarded-qualifiers',
+    '-Wno-error=suggest-attribute=format',
```

Comment:
I've added `-Wsuggest-attribute=format` to the default build flags (make & meson), and fixed the one warning that created.